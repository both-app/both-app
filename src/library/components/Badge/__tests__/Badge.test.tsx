import React from 'react'
import { render } from 'react-native-testing-library'

import { Badge } from '../'

describe('Badge', () => {
  test('should display a badge with a color', () => {
    const { toJSON } = render(<Badge color="dark200">basic text</Badge>)

    expect(toJSON()).toMatchSnapshot()
  })
})
