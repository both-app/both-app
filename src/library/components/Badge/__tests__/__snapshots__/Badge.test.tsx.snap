// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge should display a badge with a color 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#0A1D2B",
      "borderRadius": 8,
      "paddingBottom": 2,
      "paddingLeft": 8,
      "paddingRight": 8,
      "paddingTop": 2,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#FFFFFF",
        "fontSize": 14,
        "fontWeight": "bold",
      }
    }
  >
    basic text
  </Text>
</View>
`;
